pkgs/native_assets_builder/analysis_options.yaml

```diff
@@ -4,6 +4,8 @@ analyzer:
   language:
     strict-raw-types: true
     strict-inference: true
+  exclude:
+    - test/data/**
```

Comment:
These do analyze cleanly afaik.

Reply:
Perhaps I hadn't run pub get in all the right places? Or does running the tests once set some things up?

(in any case, no concerns w/ reverting this change)

Reply:
Yeah, need `pub get` in all the subfolders.

The CI needs to do the same: https://github.com/dart-lang/native/blob/main/.github/workflows/dart.yaml#L52-L74